site2/docs/tiered-storage-aliyun.md

```diff
@@ -0,0 +1,242 @@
+---
+id: tiered-storage-aliyun
+title: Use Aliyun OSS offloader with Pulsar
+sidebar_label: Aliyun OSS offloader
+---
+
+This chapter guides you through every step of installing and configuring the Aliyun Object Storage Service (OSS) offloader and using it with Pulsar.
```

Comment:
I searched for the Alibaba cloud website, it more likely uses Alibaba cloud instead of Aliyun

Reply:
I think both are fine. Hadoop and Flink documents also use Aliyun.
https://ci.apache.org/projects/flink/flink-docs-stable/deployment/filesystems/oss.html
http://hadoop.apache.org/docs/current/hadoop-aliyun/tools/hadoop-aliyun/index.html